index.js

```diff
@@ -1,4 +1,28 @@
-'use strict';
+const generateSentencePermutations = (
+  wordsList,
+  startIndex,
+  wordIndex,
+  sentences,
+  generatedSentencesArr
+) => {
+  /* eslint-disable no-param-reassign */
```

Comment:
Why do we need this bypass?

Reply:
Yes, it will be fine without semicolons. I personally don't like semicolons, but it's your decision. The bypass is because the code currently mutates the sentences parameter as it is passed from generateSentences() to generateSentencePermutations and mutated in there.

Reply:
I think we should use semicolons, I think its good practice to use it

Reply:
Semicolons are not about good practice, but are more about preference. Eslint will warn you when you have made a mistake with regards to ASI. Here's a really fun video: https://www.youtube.com/watch?v=Qlr-FGbhKaI. If you prefer semicolons, I'll include the rule.

Reply:
@cdrani Sorry for late reply, since all of the other projects I am maintaining has the semicolor rule added, we should keep the rule.
Can you update the rule and we can get this merged

Reply:
@cdrani any updates on this? Are you still working on it?